### PR TITLE
[APPINT-1121] Add sample settings key in test

### PR DESCRIPTION
**Pull request by rheasunshine**
Whatup 2018 👽

### CODE REVIEW

**Comment by ouranos** [comment]
Travis build is failing

File: template/settings/settings.yml [hunk 1]
```diff
@@ -1,2 +1,2 @@
-encryption_key1: <%= ENV['encryption_key1'] %>
+encryption_key1: <%= %w(a03f2d03 f3ae 4de0 a80a 113c22e6a37b).join('-') %>
```
Comment (ouranos) [inline template/settings/settings.yml]: Could you do that in the test.yml to not break the other env :smile_cat: ?

**Comment by rheasunshine** [comment]
@ouranos @MarcoBgn updated

File: spec/spec_helper.rb [hunk 2]
```diff
@@ -16,10 +16,18 @@
...
+    with.test_framework :rspec
+    with.library :rails
+  end
+end
+
 RSpec.configure do |config|
   config.mock_with :rspec
   config.use_transactional_fixtures = true
   config.infer_base_class_for_anonymous_controllers = false
+  config.infer_spec_type_from_file_location!
```
Comment (MarcoBgn) [inline spec/spec_helper.rb]: Good Point! 👍

**Pull request merged**